semantics/language/translation/decl/tagged.k

```diff
@@ -27,7 +27,8 @@ module C-DECL-TAGGED
           => addTag(X, makeStructFieldInfo(L))
           requires L =/=K .List
           [structural]
-     rule (.K => UNDEF("CDT1", "structs cannot be empty.", "6.7.2.1") )
+    //Undefinedness occurs when there are no named members in a struct. An empty struct is a contraint Violation. 
+     rule (.K => UNDEF("CDT1", "structs must contain a named member.", "6.7.2.1") )
```

Comment:
I've changed this back to undefined, and updated the message to reflect the error more accurately. We may have to return to this once #132 is fixed.